sqldb-dynamodb/src/lib.rs

```diff
@@ -0,0 +1,528 @@
+use aws_sdk_dynamodb::{
+    model::{AttributeValue, ReturnConsumedCapacity},
+    types::Blob,
+};
+use serde_json::{json, Value};
+use std::collections::HashMap;
+use tracing::{debug, error, instrument};
+use wasmbus_rpc::{minicbor, provider::prelude::*};
+use wasmcloud_interface_sqldb::{ExecuteResult, Parameters, QueryResult, SqlDb, Statement};
+
+mod config;
+pub use config::StorageConfig;
+mod error;
+use error::DbError;
+
+/// sqldb-dynamodb capability provider implementation
+#[derive(Clone)]
+pub struct SqlDbClient {
+    dynamodb_client: aws_sdk_dynamodb::Client,
+    ld: Option<LinkDefinition>,
+}
+
+impl SqlDbClient {
+    pub async fn new(config: config::StorageConfig, ld: Option<LinkDefinition>) -> Self {
+        // TODO: let mut aliases = config.aliases.clone();
+        let dynamodb_config = aws_sdk_dynamodb::Config::from(&config.configure_aws().await);
+        let dynamodb_client = aws_sdk_dynamodb::Client::from_conf(dynamodb_config);
+        /* TODO
+        if let Some(ref ld) = ld {
+            for (k, v) in ld.values.iter() {
+                if let Some(alias) = k.strip_prefix(ALIAS_PREFIX) {
+                    if alias.is_empty() || v.is_empty() {
+                        error!("invalid bucket alias_ key and value must not be empty");
+                    } else {
+                        aliases.insert(alias.to_string(), v.to_string());
+                    }
+                }
+            }
+        }
+        */
```

Comment:
Looks like some leftover code here, I do like the alias support in the postgres provider but if this isn't needed it can be removed